src/Command/FetchHealthStatusCommand.php

```diff
@@ -25,18 +26,34 @@ class FetchHealthStatusCommand extends Command
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $configFile = __DIR__ . '/../../config/health.config.yaml';
-
-        $retriever = new FileRetriever(__DIR__ . '/../../config/endpoints.csv');
+        $endpoints = include __DIR__ . '/../../config/endpoints.php';
 
         $storage = new FileStorage(__DIR__ . '/../../_storage');
 
         $alertingChannels = $this->getAlertingChannels($configFile);
 
         $client = new Client();
 
-        foreach ($retriever->getEndpoints() as $endpoint) {
-            $response = $client->get($endpoint);
+        foreach ($endpoints as $label => $endpoint) {
 
+            $method = $endpoint['method'] ?? 'GET';
+            $options = [
+                RequestOptions::JSON => [],
+            ];
+            if (isset($endpoint['headers'])
+                && is_array($endpoint['headers'])
+                && $endpoint['headers'] === []
```

Comment:
Shouldn't this be `$endpoint['headers'] !== []`?

Reply:
ack.